docs/pages/combobox/index.mdx

```diff
@@ -99,22 +110,26 @@ function ResizeExample() {
 
   const options = fruits.map((d, i) => ({
     value: d,
-    label: d
+    label: d,
   }));
-  
-  const [fruit, setFruit] = React.useState(options[0].value);
-  console.log('fruit ', fruit)
+
+  const [fruit, setFruit] = React.useState(null);
   return (
     <>
-     <MySelect
-      value={fruit}
-      options={options}
-      onChange={setFruit}
-      placeholder="Select one..."
-    />
+      <ComboBox value={fruit} options={options} onChange={setFruit}>
+        <ComboBoxRightElement
+          children={<Icon name="chevron-down" fontSize="1.5rem" />}
+        />
+        <ComboBoxClearElement />
+        <ComboBoxInput placeholder="Select a fruit..." />
+        <ComboBoxSelectedGhost />
```

Comment:
I think Ghost can be rendered automatically. May be through prop control we can avoid it. It should always be a ghost with the same font size, position of the text input. Composition is not required here.

Reply:
Okay, I will change it, if this ghost element should always take the inputs props. We can only control it's render via prop.

Reply:
![image](https://user-images.githubusercontent.com/39694575/80667037-ac275e00-8abb-11ea-94dd-f4a57abb2b11.png)

Done. Ghost Element now receives typography & space props from Input.